packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -289,6 +290,22 @@ Please provide a valid TCP port (an integer between 0 and 65535, inclusive).
           debuggingOptions.webEnableExpressionEvaluation
               ? WebExpressionCompiler(device!.generator!, fileSystem: _fileSystem)
               : null;
+
+        bool usesDDCModuleSystem = false;
+        const String ddcModuleFormatString = '--dartdevc-module-format=';
+        for (final String flag
+            in debuggingOptions.buildInfo.extraFrontEndOptions) {
```

Comment:
If, per my other feedback, we add a top-level tool option for the ddc module format, maybe we should add a field in `BuildInfo` for it, so we can access it here.

Reply:
Good catch - I'll defer to the flag in `BuildInfo`.